src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialServiceSupport.scala

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import java.text.SimpleDateFormat
+import java.util
+import java.util.Date
+
+import com.google.api.services.sheets.v4.model.{SpreadsheetProperties, ValueRange}
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.dataaccess.{ThurloeDAO, TrialDAO}
+import org.broadinstitute.dsde.firecloud.model.Trial.{SpreadsheetResponse, TrialProject, UserTrialStatus}
+import org.broadinstitute.dsde.firecloud.model.{UserInfo, WorkbenchUserInfo}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait TrialServiceSupport extends LazyLogging {
+
+  private val dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+  private val enrollmentFormat = new SimpleDateFormat("EEE, d MMM yyyy HH:mm:ss ")
+
+  def makeSpreadsheetResponse(spreadsheetId: String): SpreadsheetResponse = {
+    SpreadsheetResponse(s"https://docs.google.com/spreadsheets/d/$spreadsheetId")
+  }
+
+  def makeSpreadsheetProperties(title: String): SpreadsheetProperties = {
+    val dateString = dateFormat.format(new Date())
+    new SpreadsheetProperties().setTitle(s"$title: $dateString")
+  }
+
+  def makeSpreadsheetValues(managerInfo: UserInfo, trialDAO: TrialDAO, thurloeDAO: ThurloeDAO, majorDimension: String, range: String)
+    (implicit executionContext: ExecutionContext): Future[ValueRange] = {
+    import scala.collection.JavaConverters._
```

Comment:
This works as expected, but I've heard that we now prefer imports together at the top for style/consistency

Reply:
I kept it local so as not to interfere with other scala-y things but will test out moving it.